articles/architecture-scenarios/implementations/mobile-api/mobile-implementation-android.md

```diff
@@ -0,0 +1,1260 @@
+---
+description: The Android implementation of the API for the Mobile + API architecture scenario
+url: /architecture-scenarios/application/mobile-api/mobile-implementation-android
+toc: true
+---
+
+# The Mobile Implementation in Android
+
+This document is part of the [Mobile + API Architecture Scenario](/architecture-scenarios/application/mobile-api) and it explains how to implement the Mobile application in Android. Please refer to the scenario for information on the implemented solution.
+
+::: note
+The full source code for the Node.js API implementation can be found in [this GitHub repository](https://github.com/auth0-samples/auth0-pnp-abc-timesheets/tree/master/timesheets-mobile/android).
+:::
+
+## 1. Set Up the Application
+
+### System Requirements
```

Comment:
@jerriep Yes we could do this for all implementation docs for P&P. It doesn't have to be part of this PR, we can do this as a next step. Wdyt?